### DISCUSSION TITLE
[预览]v1.12.12

### DISCUSSION THREAD

**Discussion by the1812** [the1812/Bilibili-Evolved#2053] [分享 / Share]
- 修复`直播全屏包裹`和旁边的按钮没对齐.
- 修复`高分辨率图片`导致动态头像框在 Firefox 中显示异常. (#2030)
- `播放器触摸支持`适配播放器布局变动. (#2042)
- 夜间模式更新. (#1692, #2020, #2031)
- 快捷键中进度调整失效的问题: 尝试对查询 `video` 的地方附加了对 `bwp-video` 的查询. (#1931)
> 根据 @binsee 分享的播放器源码 (https://github.com/the1812/Bilibili-Evolved/issues/1931#issuecomment-878367037), `bwp-video` 应该是 Wasm 播放器的功能灰度, 虽然我还是没有被灰度到, 但先试试看 `bwp-video` 是否会把调用转发给内部的 `video` 吧

- `简化直播间`支持屏蔽标题栏活动, 只留下最基本的信息. (#2019)

<div>

<img src="https://user-images.githubusercontent.com/26504152/126054673-b76b23a4-b2a6-4978-800b-abe4e5124b16.png">
<img src="https://user-images.githubusercontent.com/26504152/126054663-62cb7d83-bad0-42f4-bee2-e30d32037e90.png">

</div>

<details>
<summary><strong>v1.12.11</strong></summary>

- 修复`瓜子换硬币`失效. (#1988)
- 修复`专栏图片导出`的命名出现`-1`. (#1973)
- 修复直播页面中`快速收起评论区`等动态功能失效. (#1900)
- 进一步完善专栏的夜间模式. (#1991)
- 修复`收起未关注人消息`打开时, `自定义顶栏`的消息数目仍然包含未关注人. (#1985)
- 分区数据更新, 影响包括`自定义顶栏`和`简化首页`. (#1511, #1918)
- `自定义顶栏`增加`番剧`入口. (#1938)

</details>

<hr /><em>This discussion was created from the release <a href='https://github.com/the1812/Bilibili-Evolved/releases/tag/v1.12.12-preview'>[预览]v1.12.12</a>.</em>

**Comment by adfnkjasd** [comment]
更新了还是有一些问题
![QQ截图20210718121859](https://user-images.githubusercontent.com/39746646/126055541-e48d2f6f-ec33-4672-9043-4e0623967cda.jpg)
![QQ截图20210718121850](https://user-images.githubusercontent.com/39746646/126055544-9467c3ae-d881-46f3-8c89-b99474aa9dd0.jpg)
浏览器版本：Chrome 版本 91.0.4472.164（正式版本） （64 位）
播放器版本：
![QQ截图20210718122031](https://user-images.githubusercontent.com/39746646/126055572-7adea0b9-0a1a-4b8c-8bd8-6adf1871faa9.jpg)
脚本版本：[预览]v1.12.12

**Reply by adfnkjasd** [reply]
已清除cookies和dns缓存 现存问题就是
1，快捷键失效 j长跳跃之类的
2，夜间模式非全屏播放器有白边和网页有白块未覆盖
3，夜间模式网页全屏顶栏会一直存在

**Reply by the1812** [reply]
你这些是3.0播放器的问题 我还没修
（全世界就我还不是3.0播放器.jpg

**Reply by adfnkjasd** [reply]
太怪了，准备放弃思考.jpg
原谅我没看更新issues，那就继续等啦
